README.md

```diff
@@ -1,121 +1,449 @@
-# ILP/STREAM
-> Interledger Transport Protocol for sending multiple streams of money and data over ILP.
+# Overview
 
-[![NPM Package](https://img.shields.io/npm/v/ilp-protocol-stream.svg?style=flat)](https://npmjs.org/package/ilp-protocol-stream)
```

Comment:
Can we keep these badges here, and maybe add a Table of Contents at the top too?

Reply:
Added